tests/topotests/bgp_rpki_topo1/r2/rpki_prefix_table.json

```diff
@@ -0,0 +1,18 @@
+{
+  "prefixes":[
+    {
+      "prefix":"62.109.128.0",
```

Comment:
Please use private (or documentation) ranges and private ASNs.

Reply:
done